exercises/practice/word-search/tests/word_search.cairo

```diff
@@ -0,0 +1,734 @@
+
+    assert(results.len() == 1, 'Expected to find 3 words');
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "coffee", start_col: 2, start_row: 1, end_col: 7, end_row: 1, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_a_left_to_right_word_in_a_two_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "tclojurerm"
+    ];
+
+    let wordsToSearchFor = array!["clojure"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 1, 'Expected to find 3 words');
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "clojure", start_col: 2, start_row: 2, end_col: 8, end_row: 2, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_a_left_to_right_word_in_a_three_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "camdcimgtc",
+        "jefblpepre",
+        "clojurermt"
+    ];
+
+    let wordsToSearchFor = array!["clojure"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 1, 'Expected to find 3 words');
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "clojure", start_col: 1, start_row: 3, end_col: 7, end_row: 3, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_a_left_to_right_word_in_a_ten_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+    ];
+
+    let wordsToSearchFor = array!["clojure"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 1, 'Expected to find 3 words');
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "clojure", start_col: 1, start_row: 10, end_col: 7, end_row: 10, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_that_left_to_right_word_in_a_different_position_in_a_ten_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+    ];
+
+    let wordsToSearchFor = array!["clojure"];
+
+    let results = search(grid, wordsToSearchFor);
+
+    assert(results.len() == 1, 'Expected to find 3 words');
+    assert_eq!(
+        results[0],
+        @SearchResult { word: "clojure", start_col: 1, start_row: 10, end_col: 7, end_row: 10, }
+    );
+}
+
+#[test]
+#[ignore]
+fn Should_locate_a_different_left_to_right_word_in_a_ten_line_grid() {
+    #[cairofmt::skip]
+    let grid = array![
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "fortranftw",
+        "alxhpburyi",
+        "jalaycalmp",
```

Comment:
This and the next row should be switched, see [canonical data](https://github.com/exercism/problem-specifications/blob/main/exercises/word-search/canonical-data.json#L244)